proxy/proxy.go

```diff
@@ -189,10 +189,16 @@ func CreateStriveProxy(listen string, GGStriveAPIURL string, PatchedAPIURL strin
 		getNews = proxy.HandleGetNews
 	}
 
+	// Make a random request to initialize TLS connection
+	resp, _ := client.Post(GGStriveAPIURL+"sys/get_env", "application/x-www-form-urlencoded", bytes.NewBuffer([]byte("data=9295a0a002a5302e302e360391cd0100")))
```

Comment:
Mixed feelings about this one, since it could end up being a waste if GGST takes a long time to load or if the user delays entering the game. Plus it creates a double-call to get_env. 

Assuming get_env is a static request, I think we'd want to cache the response, so when the user enters the game, not only do they have an active connection (assuming they enter the game <90 seconds of Totsugeki launching), they also get the first response for free.

Reply:
get_env is static for me at least. Neither request nor response body changes.
I added a new flag `unsafe-cache-env` to control if totsugeki should make that request and cache it.